src/service/fixed-layer.js

```diff
@@ -608,9 +608,9 @@ export class FixedLayer {
       return this.fixedLayer_;
     }
     const doc = this.ampdoc.win.document;
-    const bodyId = doc.body.id;
-    this.fixedLayer_ = doc.createElement('body');
-    this.fixedLayer_.id = bodyId || '-amp-fixedlayer';
+    this.fixedLayer_ = doc.body.cloneNode();
+    this.fixedLayer_.id += ' -amp-fixedlayer';
```

Comment:
Yeah, I can see that. But that breaks the previous fix. All `#bodyid .x` will be broken by this. It should be `fixedLayer_.id = doc.body.id || '-amp-fixedlayer'`. Or, tbh, we can simply kill `-amp-fixedlayer` and always use body.id.